packages/perseus-core/src/parse-perseus-json/perseus-parsers/categorizer-widget.ts

```diff
@@ -11,8 +11,8 @@ import {defaulted} from "../general-purpose-parsers/defaulted";
 
 import {parseWidget} from "./widget";
 
+import type {CategorizerWidget} from "../../data-schema";
```

Comment:
It's a two-way door so we don't have to make the change in this PR, but now that we have a folder for each widget from my changes I wonder if we could put individual widget parsers in those folders to better colocate widget logic.

Reply:
I think the proposed change might make sense. But really, the parsers are most tightly coupled to the widget options types in data-schema.ts. Unless we also move those types out of data-schema.ts into the individual widget folders, I'm inclined to keep this as-is for now.